src/lib/state/multicall.tsx

```diff
@@ -10,9 +12,34 @@ export const store = createStore(reducer)
 
 export default multicall
 
+function getBlocksPerFetchForChainId(chainId: number | undefined): number {
+  switch (chainId) {
+    case SupportedChainId.ARBITRUM_ONE:
+      return 15
```

Comment:
Omit this line - intentionally fall through to codify that they will use the same value.